tests/pfc/files/configs/pfc.py

```diff
@@ -0,0 +1,399 @@
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.reboot import logger
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.ixia.ixia_helpers import IxiaFanoutManager, get_location
+
+from tests.common.ixia.common_helpers import get_vlan_subnet, \
+    get_addrs_in_subnet
+
+###############################################################################
+# Imports for Tgen and IxNetwork abstract class
+###############################################################################
+
+from abstract_open_traffic_generator.port import Port
+from abstract_open_traffic_generator.config import Options
+from abstract_open_traffic_generator.config import Config
+
+from abstract_open_traffic_generator.layer1 import\
+    Layer1, OneHundredGbe, FlowControl, Ieee8021qbb
+
+from abstract_open_traffic_generator.device import\
+     Device, Ethernet, Ipv4, Pattern
+
+from abstract_open_traffic_generator.flow import\
+    DeviceTxRx, TxRx, Flow, Header, Size, Rate,\
+    Duration, FixedSeconds, PortTxRx, PfcPause, EthernetPause, Continuous
+
+from abstract_open_traffic_generator.flow_ipv4 import\
+    Priority, Dscp
+
+from abstract_open_traffic_generator.flow import Pattern as FieldPattern
+from abstract_open_traffic_generator.flow import Ipv4 as Ipv4Header
+from abstract_open_traffic_generator.flow import Ethernet as EthernetHeader
+from abstract_open_traffic_generator.port import Options as PortOptions
+
+def calculate_priority_vector(v) :
+    s = 0
+    for i in range(8)  :
+        if v[i] != '0' :
+           s += 2**(7 - i)
+           print(i)
+    return "%x"%(s)
+
+def lossless_iteration_list (lst) :
+    retval = [[x] for x in lst]
+    if (len(lst) > 1):
+        retval.append(lst)
+    return retval
+
+def base_configs(conn_graph_facts,
+                 duthost,
+                 lossless_prio_dscp_map,
+                 one_hundred_gbe,
+                 start_delay,
+                 traffic_duration,
+                 pause_line_rate,
+                 traffic_line_rate,
+                 pause_frame_type,
+                 frame_size,
+                 serializer) :
+
+    for config in one_hundred_gbe :
+
+        delay = start_delay * 1000000000.0
```

Comment:
I would rename start_delay to start_delay_secs and name delay as delay_nsec. Also create a global variable NSECS_IN_SEC for 1000000000.0, or better, create a function sec_to_nsec() for the conversion.

Reply:
I have changed the name of variable/fixture and wrote a function as given in the suggestion.